### ISSUE TITLE
[1.16.5 Opotato] Crash due to null in client registry of "Chunk Render Types"

### ISSUE THREAD

**Issue by NighthowlerStudios** [MCTeamPotato/Kasualix-Issue-Tracker#153]
### Specific mod for which you open this issue

Opotato (uses Kaleidio's Battle Pack for conflict)

### MC version & mod version & loader version

1.16.5

### Latest log

https://pastebin.com/TVEQe9Ze

### Steps to reproduce your issue

Very strange crash that tells nothing about itself in the trace, but gets fixed if Opotato is disabled.  Does this mod really include something that affects ChunkType registry?  If so I'm hunting to disable that mixin if possible.

**Comment by Kasualix** [comment]
OptiFine not supported. We are including https://www.curseforge.com/minecraft/mc-mods/rendertypecache to ensure incompatibility with it.

**Comment by NighthowlerStudios** [comment]
That feels like an extreme thing to do.  Do you by chance have a working, stable, shaderpack solution for 1.16.5 that allows my pack to continue benefitting from shaderpacks with Rubidium?  Or am I forced to remove Opotato?

**Comment by Kasualix** [comment]
Hmm sorry i failed to understand
Is embeddium & oculus & bsl/complementary not working for you?

**Comment by NighthowlerStudios** [comment]
Indeed, last time I tested Oculus on Forge 1.16.5, it breaks with Flywheel (Create) and causes entire chunks of the world to stop rendering.  That test was two years ago however (2023, I see there's a 2024 patch), and Rubidium does work, I will test Oculus again today to see if it improved.

**Comment by NighthowlerStudios** [comment]
I see!  It seems they've done a lot of work in 2024, it is definitely in a better state than it was for me in 2023.  I'll let my pack release with oculus then.  Thank you.
